camel-sender-demo/pom.xml

```diff
@@ -14,8 +14,8 @@
 
     <properties>
 
-        <camel-quarkus.version>2.6.0</camel-quarkus.version>
-        <camel-version>3.8.0</camel-version>
+        <camel-quarkus.version>2.7.0</camel-quarkus.version>
+        <camel-version>3.14.1</camel-version>
```

Comment:
@pilhuhn I made these changes based on [this file](https://github.com/apache/camel-quarkus/blob/2aa6d4388bad851d7205424ef746adba616e1465/pom.xml#L42-L43) from the camel-quarkus repository.

Reply:
That most likely has no impact at all (compatible change).